api/capi/doc/nnstreamer_capi.md

```diff
@@ -0,0 +1,315 @@
+# nnstreamer capi
+
+
+An application which require a data stream pipeline with deep learning model filter can be constructed and assembled easily with nnstreamer c-api.
+
+The main features of nnstreamer-capi include:
+
+- **Construction of data pipeline based on [gstreamer](https://gstreamer.freedesktop.org/)**
+
+  You can compose the data stream pipeline through nnstreamer-capi with various elements which included at gstreamer.
+
+- **[Single-shot](#singleshotapi) & [Pipeline](#pipelineapi) API**
+  
+  There are 2 types of nnstreamer-capi. Single-shot API is useful to get the output result of the model file without the composition of the data pipeline.
```

Comment:
Single-shot API is useful to get the output result of the model file without the composition of the data pipeline.

==>

Single API is useful for simple usage scenario of neural network models. It allows to invoke a neural network model with a single instance of input data for the model directly. It is useful if you have the input data pre-processed  with the application itself and there is no complex interactions between neural network models, data processors, or data stream paths.

Reply:
it's applied